### PR TITLE
Add campaign tracking parameters for all links within Matomo app that link to matomo.org

### PR DESCRIPTION
**Pull request by bx80**
### Description:

This PR is a companion change to https://github.com/matomo-org/matomo/pull/21375

It changes links to matomo.org to include campaign parameters to track the location in the application from which the link was followed.

It depends on changes in the parent PR and should only be merged along with it.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: vue/src/AdminPage/AdminPage.vue [hunk 1]
```diff
@@ -125,10 +125,9 @@ export default defineComponent({
       })}`;
     },
     logImporterNoteText() {
-      const link = 'https://github.com/matomo-org/matomo/tree/master/tests#testing-data';
       return translate(
         'VisitorGenerator_LogImporterNote',
-        `<a href="${link}" rel="noreferrer noopener" target="_blank">`,
+        externalLink('https://github.com/matomo-org/matomo/tree/master/tests#testing-data'),
```
Comment (AltamashShaikh) [inline vue/src/AdminPage/AdminPage.vue:130]: @bx80 No parameter is appended here

File: vue/src/AdminPage/AdminPage.vue [hunk 2]
```diff
@@ -110,10 +111,9 @@ export default defineComponent({
   },
   computed: {
     cliToolUsageText() {
-      const link = 'http://developer.matomo.org/guides/piwik-on-the-command-line';
       return translate(
         'VisitorGenerator_CliToolUsage',
-        `<a rel="noreferrer noopener" target="_blank" href="${link}">`,
+        externalLink('http://developer.matomo.org/guides/piwik-on-the-command-line'),
```
Comment (AltamashShaikh) [inline vue/src/AdminPage/AdminPage.vue:116]: Same here

**Review by AltamashShaikh — CHANGES_REQUESTED** [review]
Left 2 comments to check

File: vue/src/AdminPage/AdminPage.vue (hunk 2 above)
Comment (bx80) [reply vue/src/AdminPage/AdminPage.vue:116]: The parameters are being added in my tests:

image

File: vue/src/AdminPage/AdminPage.vue (hunk 1 above)
Comment (bx80) [reply vue/src/AdminPage/AdminPage.vue:130]: This is a github domain so campaign parameters won't be added.

File: vue/src/AdminPage/AdminPage.vue (hunk 2 above)
Comment (AltamashShaikh) [reply vue/src/AdminPage/AdminPage.vue:116]: @bx80 It was cache its shows up now

**Review by AltamashShaikh — APPROVED** [review]

**Pull request merged**